package/build_controller.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+STAGE_DIR=stage
+
+machine=$(uname -m)
+echo "Machine hardware architecture is \"$machine\""
+
+if [ "$machine" == "x86_64" ]; then
```

Comment:
NV only run unit-test in x86_64 env? how about arm?

Reply:
In SLSA we use emulated ARM64 build, in which unit test is very slow, so we only run unit test on amd64.  We will revisit once GitHub's arm64 node becomes generally available.

Reply:
We should decouple the testing from the building script - that way it is not cross-emulated.
A new workflow for tests can be created, which simply run tests at both `push`/`pull_request` events.